src/gui/editors/AutomationEditor.cpp

```diff
@@ -2220,6 +2222,7 @@ AutomationEditorWindow::AutomationEditorWindow() :
 	setFocusPolicy( Qt::StrongFocus );
 	setFocus();
 	setWindowIcon( embed::getIconPixmap( "automation" ) );
+	setAcceptDrops( true );
```

Comment:
Nitpicking here... it appears we generally link the toolbar as well... 

https://github.com/LMMS/lmms/blob/master/src/gui/editors/BBEditor.cpp#L56